### PR TITLE
Implement [ACR] The Animus

### PR DESCRIPTION
**Pull request by grimreap124**
https://scryfall.com/card/acr/69/the-animus

### CODE REVIEW

File: Mage.Sets/src/mage/cards/t/TheAnimus.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer(source.getControllerId());
+        UUID targetId = getTargetPointer().getFirst(game, source);
+        if (controller == null || targetId == null) {
+            return false;
+        }
+
+        Card card = game.getCard(targetId);
```
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/t/TheAnimus.java:94]: null check

File: Mage.Sets/src/mage/cards/t/TheAnimus.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        controller.moveCards(card, Zone.EXILED, source, game);
+        card.addCounters(CounterType.MEMORY.createInstance(), source.getControllerId(), source, game);
```
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/t/TheAnimus.java:96]: If move the add counter. Or even better: use retain - make sure it’s in same zone before put counter on it:

IMG_0512

**Review by JayDi85 — APPROVED** [review]

File: Mage.Sets/src/mage/cards/t/TheAnimus.java (hunk 2 above)
Comment (grimreap124) [reply Mage.Sets/src/mage/cards/t/TheAnimus.java:102]: Hopefully the check I added is what you wanted

File: Mage.Sets/src/mage/cards/t/TheAnimus.java [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+            return false;
+        }
+
+        Card card = game.getCard(targetId);
+        if (card == null) {
+            return false;
+        }
+        controller.moveCards(card, Zone.EXILED, source, game);
+        if (game.getState().getZone(card.getId()) != Zone.EXILED) {
+            return false;
```
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/t/TheAnimus.java:100]: Must return true here. Basic logic in most use cases:
* if effect something changed in game state then return true;
* If nothing changes then return false.

**Comment by xenohedron** [comment]
see comments on the commit, looks like this wasn't quite ready for being merged

**Pull request merged**